rest_api/v2/components/index.yml

```
@@ -2394,6 +2595,115 @@ schemas:
                 $ref:  '#/schemas/CustomFieldBusinessesCount'
               description:
                 $ref:  '#/schemas/CustomFieldDeleteDescription'
+  AttributeType:
+    description: Type of the Attribute
+    type: string
+    enum:
+      - URL
+      - REPEATABLE_URL
+      - BOOL
+      - ENUM
+      - REPEATABLE_ENUM
+  AttributeGmbId:
+    description: Google mu Business Id of the attribute
+    type: string
+    example: 'gcid:restaurant'
+  AttributeName:
+    description: Name of the attribute
+    type: string
+    example: 'Delivery'
+  AttributeOrder:
+    type: integer
+    description:  Order in the which in the Attribute will be displayed on the Business Edit view of the Partoo App
+    example: '1'
+  Attribute:
+    type: object
+    properties:
+      type:
+        $ref: '#/schemas/AttributeType'
+      gmb_id:
+        $ref: '#/schemas/AttributeGmbId'
+      name:
+        $ref: '#/schemas/AttributeName'
+      order:
+        $ref: '#/schemas/AttributeOrder'
+  AttributesGroupName:
+    type: string,
+    description: Name of the attribute group
+    example: "Services disponibles"
+  AttributesGroupOrder:
+    type: Integer
+    description: Order in the which in the Attribute Group will be displayed on the Business Edit view of the Partoo App
+    example: 5
+  AttributesGroup:
+    type: object
+    properties:
+      attributes:
+        type: array
+        description: List of Google Attributes by Group
+        items:
+          $ref: '#/schemas/Attribute'
+      name:
+        $ref: '#/schemas/AttributesGroupName'
+      order:
+        $ref: '#/schemas/AttributesGroupOrder'
+  AttributesGroupList:
+    type: array
+    description: List of Google Attributes
```

```suggestion
    description: List of Google Groups
```
et peut-être une petite phrase pour expliquer ce que sont les groupes